index.js

```diff
@@ -39,6 +39,48 @@ let ops = [
   "OR"
 ];
 
+// Array of usable functions
+let funs = [
+  "CEILING",
+  "FLOOR",
+  "ABS",
+  "TRUNCATE",
+  "EXP",
+  "SQRT",
+  "LN",
+  "LOG",
+  "ceiling",
+  "floor",
+  "abs",
+  "truncate",
+  "exp",
+  "sqrt",
+  "ln",
+  "log",
+  "NOT",
+  "not",
+];
+
+// Array of functions in the form x.func() 
```

Comment:
I am not sure what you mean by "in the form x.func()".  How do these differ from the list above?

Reply:
Fixed